packages/cli/test/dev/integration-5.test.ts

```diff
@@ -523,10 +523,13 @@ describe('[vercel dev] ESM serverless functions', () => {
     '[vercel dev] ESM .js type=commonjs',
     testFixtureStdio(
       'esm-js-nodejs-no-module',
-      async (testPath: any) => {
-        // bad gateway
-        // require() of ESM Module
```

Comment:
This test seemed to indicate that 500 was expected. I don't think we should change the behavior because thats not how Node.js works, right?

Reply:
For example

```
npm init -y
npm add leap-year
echo "var isLeapYear = require('leap-year')" > file.js
node file.js

var isLeapYear = require('leap-year')
                   ^
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/styfle/Code/foo/leapyear/node_modules/.pnpm/leap-year@4.0.0/node_modules/leap-year/index.js from /Users/styfle/Code/foo/leapyear/file.js not supported.
```

Reply:
You are correct about that actually. Might need to reconsider this approach then <picture data-single-emoji=":thinkies:" title=":thinkies:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/thinkies/d164ba479fed9c0d.png" alt=":thinkies:" align="absmiddle"></picture>